docs/cli/init.md

```diff
@@ -25,5 +31,7 @@ initialize a [source-git repo](/source-git).
       source-git repo.
 
     Options:
-      -f, --force  Reset config to default if already exists.
-      -h, --help   Show this message and exit.
+      --force-precommit    Automatically create an empty pre-commit configuration file if missing.
+      --without-precommit  Skip adding pre-commit hook to pre-commit configuration file.
+      -f, --force          Reset config to default if already exists.
+      -h, --help           Show this message and exit.
```

Comment:
This part should be automated, but I guess the change won't hurt, it will just open another PR.